<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="eve_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>eveg.gif</normaloff>eveg.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="sang_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>strange.gif</normaloff>strange.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ggo_btn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>gogog.gif</normaloff>gogog.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pika_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pikag.gif</normaloff>pikag.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="ohoh_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Herculanum</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>오늘의 포켓몬을 고르세요? 멀까요?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
